src/__tests__/inspection.test.ts

```diff
@@ -55,14 +55,28 @@ it('reports too few lines with disallowed one-liners.', () => {
   ]);
 });
 
+it('reports no errors on any message when body check is disabled.', () => {
+  const inputs = input.parseInputs('', '', 'true', '', 'true').mustInputs();
```

Comment:
```suggestion
  const inputs = input.parseInputs('', '', '', '', 'true').mustInputs();
```

If I'm not mistaken, you don't have to explicitly set `allowOneLinersInput`?